test/activity-repository-test.js

```diff
@@ -11,9 +11,9 @@ describe('Activity', () => {
   beforeEach(() => {
     activity = new Activity(activityTestData);
     user1 = new User(userTestData[0]);
-    user2 = new User(userTestData[1]);
```

Comment:
did we need to change some tests?

Reply:
Yeah, I still have a little ways to go in the testing realm, but it should be as simple as renaming some methods. I might be able to cut down on some tests as well, or at least use some nested `describe` blocks so things are a bit more readable!